data/hai/unfettered side missions.txt

```diff
@@ -316,7 +401,7 @@ mission "Hai Legacy 3"
 		on kill
 			fail "Hai Legacy 3"
 	on visit
-		dialog "You have not beaten all challenging ships yet. If they have not shown up it is probably because you have cheated and brought more ships into the fight."
+		dialog "You have not beaten all challenging ships yet. If they have not shown up it is probably because you have cheated and brought more ships into the fight, load a save and try again if you want to finish this mission."
```

Comment:
```suggestion
		dialog "You have not beaten all challenging ships yet. If they have not shown up it is probably because you have cheated and brought more ships into the fight. Load an earlier save and try again if you want to finish this mission."
```